src/main.rs

```diff
@@ -188,10 +188,29 @@ fn aot_main(buf: &str, opt: Opt, output: &Path, color: ColorChoice) -> Result<()
 
 fn handle_warnings(warnings: VecDeque<CompileWarning>, file_db: &Files, color: ColorChoice) {
     WARNINGS.fetch_add(warnings.len(), Ordering::Relaxed);
+    #[cfg(not(feature = "salty"))]
+    let warn = "warning";
+    #[cfg(feature = "salty")]
+    let warn = {
+        use rand::Rng;
+
+        let msgs = [
+            "you sure this is a good idea?",
+            "this is probably fine",
+            "if your majesty in their wisdom thinks this is a good idea, far be it from me to argue",
+            "this is why the C standards committee has nightmares",
+            "how does this make you feel?",
+            "I'm checking some blueprints, and I think... Yes, right here. You're definitely going the wrong way",
+            "I don't want to tell you your business, but if it were me, I'd leave that thing alone",
+        ];
+        let mut rng = rand::thread_rng();
+        msgs[rng.gen_range(0, msgs.len())]
```

Comment:
Rather than picking a random number every time, it'd possibly be better to pick a random starting point, a direction or prime number increment, and then just step the messages in order. This way you won't get four times the same message in a sequence because RNGeezus felt like it.